pkg/component/watch_test.go

```diff
@@ -55,3 +67,291 @@ func TestIsRegExpMatch(t *testing.T) {
 	}
 
 }
+
+func setUpFabric8AnalyticsLicenseAnalysisComponentSrc(componentName string, requiredFilePaths []FileProperties) (string, map[string]FileProperties, error) {
```

Comment:
This seems like a really long function name... could you make it shorter?

Reply:
Ack